### DISCUSSION TITLE
Why branch length is always wrong?

### DISCUSSION THREAD

**Discussion by YuChrming** [hahnlab/CAFE5#105] [Q&A] [answered]
I used mcmctree to get my timetree ,and the result looks like `((((MHUC: 6.0176, (MHUA: 4.8533, MHUB: 4.8533) : 1.1644):1.6189, MBAC: 7.6366) : 1.4433, ((((GDDH: 4.2891, HONE: .4.2891) : 1.1111, ((GALA: 3.3719, MSIE: 3.3719) : 0.6001, HFTH: 3.9720): 1.4283): 0.9826, MSYL: 6.3828) : 1.9412, MPRU: .8.3239): 0.7560): 5.6622, PBET: 14.7421);`

But it shows `Invalid branch length set for 3.3719` when I ran cafe. How it causes or say mcmctree is not recommended?By the way, the species I used are in one genus, is this the reason?

My gene_families.txt
![image](https://user-images.githubusercontent.com/92568144/204228648-ef0ecb23-574c-43e0-ba86-9ee8e11f0db5.png)

**Comment by benfulton** [comment, marked as answer]
CAFE does not handle spaces in trees, unfortunately. Go through and remove all the spaces and it should be recognized as a valid tree.

**Reply by YuChrming** [reply]
Thank you very much for your reply! I remove all the spaces and execute again. But met an Error 
![image](https://user-images.githubusercontent.com/92568144/204459568-adf530e3-2c88-404f-a576-bf44e819306a.png)
Is the value not true ?

**Comment by benfulton** [comment]
That indicates that no value could be calculated for the families and tree you selected. As the message states, you can try removing the families with the largest deltas to see if a value can be calculated then.
